### PR TITLE
Move commands to appropriate mode files

### PR DESCRIPTION
**Pull request by jmercouris**
Relatively simple change, moves commands that rely on a mode, to the file where the mode itself is defined.

### CODE REVIEW

**Comment by jmercouris** [comment]
I forgot, we had already discussed this as well on Monday...

**Comment by jmercouris** [comment]
Specifically in the context of 'why is list-buffers defined in base-mode???'

File: source/browser.lisp [hunk 1]
```diff
@@ -58,7 +58,6 @@ This slot is mostly meant to clean up the thread if necessary.")
    (password-interface (make-password-interface)
                        :export nil)
    (messages-content '()
-                     :export nil
```
Comment (Ambrevar) [inline source/browser.lisp:61]: I think this should remain unexported: this is an implementation detail, we don't want the user to access it directly.

Or, at least, just define a `reader`.

**Comment by Ambrevar** [comment]
Thanks!

File: source/browser.lisp (hunk 1 above)
Comment (jmercouris) [reply source/browser.lisp:61]: A good call, a reader it should be!

Comment (Ambrevar) [reply source/browser.lisp:61]: Done with 12776890f0ad6312126e3c3b46bc28c0a2ddb605.

Comment (jmercouris) [reply source/browser.lisp:61]: Thank you!

**Pull request merged**